forge/src/main/java/net/dodogang/plume/ash/registry/forge/RegistryBatchImpl.java

```diff
@@ -50,9 +51,11 @@ public RegistryBatchImpl(RegistryKey<Registry<T>> registryKey, String modId) {
 
     @Override
     protected void registerImpl() {
-        IEventBus modEventBus = ModEventBus.getModEventBus(modId).orElseThrow(() -> new IllegalStateException(
-                "Attempted to register BatchedRegister before registering a ModEventBus for modid '" + modId + "'.")
-        );
+        IEventBus modEventBus = ModEventBus.getModEventBus(modId).orElseThrow(
+            ()
+                -> new IllegalStateException(
```

Comment:
Having the lambda parameters on a different line than the inside of the lambda is ugly. Why does this even exist?

Reply:
can I shoot it